sda-commons-server-opentelemetry/src/main/java/org/sdase/commons/server/opentelemetry/jaxrs/JaxrsPathUtil.java

```diff
@@ -0,0 +1,22 @@
+package org.sdase.commons.server.opentelemetry.jaxrs;
+
+import java.io.File;
+
+public final class JaxrsPathUtil {
+  private JaxrsPathUtil() {}
+
+  public static String normalizePath(String path) {
+    // ensure that non-empty path starts with /
+    if (path == null || "/".equals(path)) {
+      path = "";
+    } else if (!path.startsWith("/")) {
+      path = File.separator + path;
```

Comment:
Using `File.separator` will probably not work on Windows machines. We should use the same separator consistently here.

Reply:
True, updated